packages/rx_bloc_cli/example/docs/auth_matrix.md

```diff
@@ -0,0 +1,257 @@
+```jsonc
+{    
+ // ...payload specific properties,   
+ "type": String, //The type of the payload. This is used to determine the type of the payload when serialising/deserializing.  
+}   
+```   
+### Response
+
+**Body:**
+```jsonc
+{
+ // The unique auth matrix transaction id. It's used in the following payload requests to ensure the chain of requests.
+ "transactionId": "1",
+
+ // The draft document ids created by the initial auth matrix request. Once the user completes all authentication methods successfully, the documents will be marked as signed.
+ "documentIds": List<int>,
+
+  // The security token, which ensures the chain of auth matrix requests. It's used in the following payload requests to ensure the chain of requests.      
+ "securityToken": String, 
+
+  // The ISO 8601 expiration date of the auth matrix request.  
+ "expires": String,
+
+  /// The of auth matrix method (predefined enum) to be used for the following request.
+ "authMethod": String,
+
+ // Dynamic additional data along with the required response properties. Create custom AuthMatrixResponsePayload implementation for each case.  
+ "payload": null 
+}   
+```
+ ---   
+## POST /api/auth-matrix/{id}
+
+### Request
+
+**Body:**
+```jsonc
+{    
+  "securityToken": String,
+  "payload": {
+    // ...payload specific properties,
+    "type": "pinBiometric"
+  }
+}   
+```   
+### Response
+
+**Body:**
+```jsonc
+{
+ // The unique auth matrix transaction id. It's used in the following payload requests to ensure the chain of requests.
+ "transactionId": "1",
+
+ // The draft document ids created by the initial auth matrix request. Once the user completes all authentication methods successfully, the documents will be marked as signed.
+ "documentIds": List<int>,
+
+  // The security token, which ensures the chain of auth matrix requests. It's used in the following payload requests to ensure the chain of requests.      
+ "securityToken": String, 
+
+  // The ISO 8601 expiration date of the auth matrix request.  
+ "expires": String,
+
+  /// The of auth matrix method (predefined enum) to be used for the following request.
+ "authMethod": String,
+
+ // Dynamic additional data along with the required response properties. Create custom AuthMatrixResponsePayload implementation for each case.  
+ "payload": null 
+}   
+```    
+
+# Authentication action
+To add a new authentication method, such as `change password`, first, add the new method to the AuthMatrixAction enum. 
+Then, extend the AuthMatrixPayloadRequest class to create a payload class for the new method, 
+incorporating necessary payload fields and implementing methods for JSON serialization and equatability.
+
+#### Step 1: Add a new case in the AuthMatrixAction enum
+New case in the AuthMatrixAction needs to be added for each new authentication method.
+```dart
+enum AuthMatrixAction {
+  // the other actions...
+  changePassword,
+}
+```
+
+#### Step 2: Add a new case in the AuthMatrixActionExtension
```

Comment:
I can't find any reference to `AuthMatrixActionExtension` anymore

Reply:
The documentation is updated